app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  mount_uploader :avatar, AvatarUploader
+  validates :username, :firstname, :lastname, :email, :bio, :presence => true
```

Comment:
Minor suggestion, not a blocker: I would generally recommend that this type of change, and any corresponding tests, show up in their own PR since nothing on this line is directly related to the Carrierwave integration.